__TESTS__/unit/actions/Flag.test.ts

```diff
@@ -0,0 +1,232 @@
+import TransformableImage from '../../../src/transformation/TransformableImage';
+import CloudinaryConfig from "../../../src/config/CloudinaryConfig";
+import Flag, {animated} from "../../../src/actions/flag/Flag";
+import expectESMToMatchDefault from "../../TestUtils/expectESMToMatchDefault";
+import * as FlagESM from "../../../src/actions/flag/Flag";
+
+
+const CONFIG_INSTANCE = new CloudinaryConfig({
+  cloud: {
+    cloudName: 'demo'
+  }
+});
+
+describe('Tests for Transformation Action -- Flag', () => {
+  it('Expects ESM to match Default', () => {
+    expectESMToMatchDefault(FlagESM, Flag);
+  });
+
+  it('Ensure namespace is correctly populated', () => {
+    expect(Flag.animated).toEqual(animated);
+  });
+
+  it('Creates a cloudinaryURL with Flag animated', () => {
+    const url = new TransformableImage()
+      .setConfig(CONFIG_INSTANCE)
+      .addFlag(Flag.animated())
+      .setPublicID('sample')
```

Comment:
We should add a test for multiple flags, these should be concatenated with a dot.

such as `fl_foo.bar.zoo`